builders/Dockerfile.mingw64

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright 2022 The Geany contributors
+# License: GPLv2
+#
+# Docker image for Geany and Geany-Plugins cross-build to Windows
+# The image contains a self-compiled Pacman to install mingw-w64
+# packages and all other dependencies necessary to build the code
+# and create a ready-use installer.
+# For more details, see build_mingw64_geany.sh where this image is used.
+#
+# Intermediate container for building pacman
+FROM debian:bullseye as build-pacman
```

Comment:
Why is that an "intermediate container"? And what does this mean exactly, i.e. what's the advantage over building pacman in the main docker container?

Reply:
The advantage is that everything necessary to build Pacman remains in the intermediate container and gets destroyed automatically once it is finished. Nothing from this container will remain in the final container except those items which you copy explicitly.
This is very common and convenient practice when building Docker images.
https://docs.docker.com/build/building/multi-stage/